### ISSUE TITLE
Could you please compile a PL dictionary?

### ISSUE THREAD

**Issue by bartosz-antosik** [Jason-Rev/vscode-spell-checker#79]
This is related to the discussion about [#20266](https://github.com/Microsoft/vscode/issues/20266#issuecomment-288860535).

I started digging into cspell-tools but then, at the very beginning, decided to ask you: Could you please compile Polish dictionary for me? You can download it [here](https://github.com/wooorm/dictionaries/tree/master/dictionaries/pl_PL), both DIC and AFF files. Beware, it's big: 300K word roots plus large affixes base.

I would love to see how does your implementation compare to hunspell.

**Comment by Jason3S** [comment]
I just now setup a package on npm:
https://www.npmjs.com/package/cspell-dict-pl_pl

See if that works for you.

**Comment by bartosz-antosik** [comment]
It seems like it has installed correctly a aside cspell, cspell-lib, cspell-trie etc. What should I put in "cSpell.language" to make it spell in polish? Seems that neither "pl_pl" nor "Polish" work.

**Comment by Jason3S** [comment]
The way I described it, Polish is always on.

If it doesn't load, you will see something like this in the VSCode Output:

```
[Error - 5:10:59 PM] Request textDocument/codeAction failed.
  Message: Request textDocument/codeAction failed with message: Unknown file format
  Code: -32603 
```

Yes, it is a poor error message.  That needs some work.

If you do a:

```sh
npm install -g cspell-dict-pl_pl
```

Then something like this can be added to your user settings:

```javascript
"cSpell.dictionaryDefinitions": [
    { "name": "Polish", "path": "/usr/local/lib/node_modules/cspell-dict-pl_pl/pl_pl.trie.gz"}
],
"cSpell.dictionaries": [
    "Polish"
],
```

You will need to change the path to match the location on your machine.

**Comment by Jason3S** [comment]
Example of what it looks like for me:

![image](https://cloud.githubusercontent.com/assets/3740137/24305432/39541f38-10bd-11e7-831a-23ab81fcbb2e.png)

I don't know what the words mean, so please forgive me if it is something strange.

**Comment by Jason3S** [comment]
I am working on a way to make this easier, but for now only edit .json is possible.

I would like to have some UI that allows you to pick which dictionaries you want active.

**Comment by bartosz-antosik** [comment]
OK, it is working fine. Thanks. I will try to perform some tests.

Is this compiled dictionary  created using all of the affixes?

**Comment by Jason-Rev** [comment]
All the affixes should be included.

**Comment by Jason3S** [comment]
I'm going to close this.  Let me know if you see any issues.
